quarkchain/cluster/master.py

```diff
@@ -1297,6 +1287,10 @@ def handle_new_root_block_header(self, header, peer):
         future_list = self.broadcast_rpc(
             op=ClusterOp.ADD_ROOT_BLOCK_REQUEST, req=AddRootBlockRequest(r_block, False)
         )
+
+        for _ in self.cluster_config.GRPC_LIST:
+            GrpcClient().set_rootchain_confirmed_block()
```

Comment:
you sure you want to use default parameters? every slave may have different host / port, right?